docs/configuration.md

```diff
@@ -580,6 +592,13 @@ Options:
           Apply fixes to resolve lint violations, but don't report on leftover
           violations. Implies `--fix`. Use `--no-fix-only` to disable or
           `--unsafe-fixes` to include unsafe fixes
+      --config <CONFIG_OPTION>
```

Comment:
It feels a bit strange that `--isolated` is in `Miscellaneous` and `--config` is not. I think I would move `--config`